orcestra_book/plans/HALO-20240923a.md

```diff
@@ -1,23 +1,23 @@
 ---
 arrival_airport: TBPB
-categories: []
+categories: [ec_under, c_meteor, c_west, c_mid, c_south]
```

Comment:
The categories `c_meteor` and `c_west` do not exist, yet. You should either need to add them to `orcestra_book/operation/flight_categories.yaml` or remove the tags.